Resources/Prototypes/Guidebook/chemicals.yml

```diff
@@ -0,0 +1,60 @@
+- type: guideEntry
+  id: Chemicals
+  name: guide-entry-chemicals
+  text: "/ServerInfo/Guidebook/Chemicals.xml"
+  children:
+  - Elements
+  - Medicine
+  - Narcotics
+  - Pyrotechnic
+  - Toxins
+  - Foods
+  - Botanical
+  - Biological
+  - Others
+  filterEnabled: True
+
+- type: guideEntry
+  id: Elements
+  name: guide-entry-elements
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Elements.xml"
+
+- type: guideEntry
+  id: Medicinal
+  name: guide-entry-medicine
+  text: "/ServerInfo/Guidebook/Medical/Medicine.xml"
```

Comment:
This entry is unused. It's a near-copy of the existing `Medicine` page but with a different ID anyway.